### PR TITLE
Can 212

### PR DESCRIPTION
**Pull request by shahab-ramzan**
camp statement history

### CODE REVIEW

File: src/pages/statement/history/[...camp].tsx [hunk 1]
```diff
@@ -16,21 +16,21 @@ const CampHistoryPage = ({ history }) => {
...
 
-export async function getServerSideProps() {
-  const reqBody = {
-    topic_num: 75,
-    camp_num: 1,
-    type: "all",
-  };
-  const res = await getCampStatementHistoryApi(reqBody);
-  const history = res || [];
+// export async function getServerSideProps() {
```
Comment (akuks) [inline src/pages/statement/history/[...camp].tsx]: Do we need this?

File: src/network/request/campStatementHistoryRequest.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default class CampStatementHistoryRequest extends Request {
...
+      "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJhdWQiOiIyIiwianRpIjoiODI2M2NmMjBlODVkMjRhY2EwOTk0MzZlMWNjMWNjNmRjMmE3NDE5YzM1OTIwN2E4ODg0YWY5OTNiNjA5NzI5Yjk4NTJhYzUwOTFmN2JlMjciLCJpYXQiOjE2NTA1MjUwMzEuMzA3MzE1LCJuYmYiOjE2NTA1MjUwMzEuMzA3MzE5LCJleHAiOjE2ODIwNjEwMzEuMjk3MjM2LCJzdWIiOiIxMjMwIiwic2NvcGVzIjpbIioiXX0.P83KeK4hvp63lgPMu64gCfMilA8TfkSWWumga7jZP59gXf3xSPi7MR1FR1M6S02oU52ElSdbkDKAHwgJGbE_siOw-VrYk-PZPJWhdUH5hYAswXSxRZ9aYB1dm6NHzDrTuyS33GKTIm1qQKnFwNGomL7BnuoTys2BU49YVZxVusyiSKRF_vXqDBb39HNsCUCGOO-ZFM3I1iYEVF3EXgg5cHyRTAasKMUFe3RcGk5xDnpls18NVazH4IX6tsQeMI-mqDxAv6XqJ3g-egLMb4kRZcr6b7TwtvXSFXPd4YPCoBQJbeBHpKuWn6QP3UlPXCszgLe4qZI00CKAzpK38V6EkvwHuG_Bflfc8aK8LF4v2Al_8gLal3rh92ZkArk8xu7-553s2ZkOloKRicLp7ckssFKzXuYS5P8RLiLKWrt9OkPAEE7w143W8vIzqKaYc-7g-xQPyrVrfvE2P6pAR7wE0SwlYvvP8pSeYsQEUZEKuHHyOrVnD-Cm8qjngUgYGL3cewSvVwfYOcXZFfN8rYECFsam-BAQ6T3p1inlbDTXVVgqnsrQFZ9oJU9R6Qu6k28WIVlA8eDfu_vQmpW-3mn2IMUoNiTvqxcQ3pt7mnMaL9BdrsjB-uTbkeFJwVn77jIi8WXrYLfM36ag9vDsZFpxmvdlORUxSKwfBefN5BRAadw"
```
Comment (akuks) [inline src/network/request/campStatementHistoryRequest.ts]: Looks like a JWT token. Do we need this?

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/network/request/campStatementHistoryRequest.ts (hunk 2 above)
Comment (shahab-ramzan) [reply src/network/request/campStatementHistoryRequest.ts]: Removed

File: src/pages/statement/history/[...camp].tsx (hunk 1 above)
Comment (shahab-ramzan) [reply src/pages/statement/history/[...camp].tsx]: removed

**Pull request merged**